include/struct.h

```diff
@@ -77,6 +77,7 @@ struct cluster {
   double vary;
   double varz;
   std::vector<dg_cell> cells;
+  std::vector<double> cells_e;
```

Comment:
`cells_e` seems a cell quantity related to `dg_cell` oblect. Would it be better to add it to `dg_cell`?

Reply:
The commit e4e6a9fac27c74356e496a80238c30caad4e71f4 adds `reco_cell` and `vector<reco_cell>` in `cluster` object in `struct.h`. 
Each `reco_cell` contains the geometrical cell informations (module, layer, id) and the reconstructed ones x(barrel), y(endcap), e, t; 
In this way we have: 
- Collection of _PMTHit_ in `dg_cell` object as `vector<dg_ps>` (1 and 2 for the two PMTs),
- _RecoHit_ in `reco_cell` object as x(barrel), y(endcap), z, e and t quantities. 
Maybe a reconstructed structure composed of (like the one of `dg_cell` with `vector<dg_ps>`) `reco_cell` with `vector<`_RecoHit_`>`, is not the best for cluster object because in principle each cluster should have only one _RecoHit_ in each cell.

Reply:
add `dg_ps1` and `dg_ps2` to `reco_cell` and remove `vector<dg_cell>` from `cluster`